src/pages/Agregator/Agregator.component.tsx

```diff
@@ -0,0 +1,162 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Paper from '@mui/material/Paper';
+import OutlinedInput from '@mui/material/OutlinedInput';
+import { BigNumber, utils } from 'ethers';
+import { useState, useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { ControlledDropdown } from '../../components/ControlledDropdown/ControlledDropdown.component';
+import { PageView } from '../../components/PageView/PageView.component';
+import { ControlledInputWithButtonPillGroup } from '../../components/ControlledInputWithButtonPillGroup/ControlledInputWithButtonPillGroup.component';
+import { baseChains, ChainEnum, chains } from '../../config/chains';
+import { TokenTypeBase } from '../../config/tokens';
+import { NameWithIcon } from '../../components/NameWithIcon/NameWithIcon.component';
+import { AggregatorInputs } from '../../constants';
+
+export const AgregatorComponent = () => {
+  const [bassetOptions, setBassetOptions] = useState<TokenTypeBase[]>();
+  const [tokenDropdownDisabled, setTokenDropdownDisabled] = useState(true);
+  const [availableBalance, setAvailableBalance] = useState<BigNumber>();
+  const [receiveAmount, setReceiveAmount] = useState('0.00');
+
+  const { watch, setValue, control } = useForm();
+  const watchChain = watch(AggregatorInputs.ChainDropdown);
+  const watchToken = watch(AggregatorInputs.TokenDropdown);
+
+  useEffect(() => {
+    const chosenChain = baseChains.find(({ id }) => id === watchChain);
+    const currentOptions = chosenChain && chosenChain.bassets;
+    setBassetOptions(currentOptions);
+    setTokenDropdownDisabled(!currentOptions);
+  }, [watchChain]);
+
+  const getTokenAvaliableAmount = () => {
+    // todo: implement
+    setAvailableBalance(utils.parseEther('81'));
+  };
+
+  useEffect(() => {
+    if (watchToken) {
+      getTokenAvaliableAmount();
+    }
+  }, [watchToken]);
+  return (
+    <form
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        maxWidth: 1120,
+        margin: '0 auto',
+      }}
+    >
+      <PageView
+        title={
+          <Box
+            sx={{
+              py: 2.5,
+              px: 3,
+            }}
+          >
+            <Typography variant="h2">Starting chain</Typography>
+          </Box>
+        }
+      >
+        <Box
+          sx={{
+            width: 310,
+            mx: 'auto',
+          }}
+        >
+          <ControlledDropdown
+            name={AggregatorInputs.ChainDropdown}
+            label="Select Network"
+            defaultValue=""
+            placeholder="Select Chain"
+            control={control}
+            options={baseChains}
+            sx={{ mb: 6 }}
```

Comment:
margins / paddings in this page seems to be a lot bigger than in the designs.
![image](https://user-images.githubusercontent.com/49481475/154270186-fe3114b8-f1ac-483e-aee4-c96aeae97ed4.png)
![image](https://user-images.githubusercontent.com/49481475/154270209-a448c5e1-0cb9-4840-bf04-2c636dc25ad1.png)

Reply:
There's no padding/margin here, it's just e growing flex container. With the button in the middle (which is not there yet) it will look fine

Reply:
Ok, got it, but the vertical spaces between inputs seems to be bigger than in the designs as well. I use 3 spacing units in the forms from the staking page and here you have 6. I think we need to agree on one value

Reply:
I think that is not necessarily true. Not everything needs to be identical, espetially considering that every input does somethin a bit different.